config/initializers/decidim.rb

```diff
@@ -15,9 +15,6 @@
      here_app_code: Rails.application.secrets.geocoder[:here_app_code]
    }
 
-  # Default app time zone
-  config.time_zone = 'Madrid'
```

Comment:
why not comment it and add a comment like:
"""
Can not force time_zone as all timestamps in DB are in UTC and ought to be updated if this config was applied.
"""